util/inc/WireCellUtil/Response.h

```diff
@@ -177,6 +181,25 @@ namespace WireCell {
 
 	};
 
+
+  /// A functional object caching gain and shape.
+  /// ICARUS warm electronics
+	class WarmElec : public Generator {
+	    const double _g, _s;
+	public:
+	    // Create warm electronics response function.  Gain is an
+	    // arbitrary scale, typically in [voltage/charge], and
+	    // shaping time in WCT system of units.
+	    WarmElec(double gain=14*units::mV/units::fC, double shaping=1.0*units::us);
```

Comment:
One real thing to maybe check: are 14 and 1.0 reasonable defaults?  If there is an overwhelming popular default for WarmElec parameters, best to use them here.

Reply:
Thanks for pointing that out. Actually I kept the same of the cold elec. I can put the values of the paper as default